scripts/tests/test_fix_dsi_studio_trk.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import os
+import tempfile
+
+from scilpy.io.fetcher import fetch_data, get_home, get_testing_files_dict
+
+# If they already exist, this only takes 5 seconds (check md5sum)
+fetch_data(get_testing_files_dict(), keys=['tracking.zip'])
```

Comment:
don't need to fetch anything or maybe you could add some dsi data in this package

Reply:
I deleted the line about the fetch, this script is (for now) for case-by-case basis and DSI data is basically unique so I think for now I would prefer just to keep it like that until someone actually test it in real life condition.